### PR TITLE
Phase 1 - Restructure Homepage

### PR DESCRIPTION
**Pull request by KimberleyCook**
As part of the tutorials website revamp I have moved the tutorials into 4 sections; Frontend, Backend, Apps, and General. See screenshot below.

I also increased the font size, based on this [issue](https://github.com/codebar/tutorials/issues/465) raised,a made the grey text a little darker for accessibility.

screencapture-0-0-0-0-4000-2021-04-12-13_27_29

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -13,3 +13,4 @@ exclude: [
   "node_modules"
 ]
 livereload: true
+exclude: [vendor]
```
Comment (KimberleyCook) [inline _config.yml:16]: Had to add this to the config to get it running locally for me.

Comment (gnclmorais) [reply _config.yml:16]: No problem in adding this, we might want to add [even more files & folders](https://github.com/jekyll/jekyll/blob/6855200ebda6c0e33f487da69e4e02ec3d8286b7/lib/site_template/_config.yml#L37-L55).

**Review by gnclmorais — APPROVED** [review]
Great change, especially the bigger font size!

**Comment by KimberleyCook** [comment]
Thanks for reviewing @gnclmorais 🎉

**Pull request merged**